.github/workflows/build_docker.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build docker
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
```

Comment:
Can you add **Inline** cache 
- https://docs.docker.com/build/cache/backends/
- https://docs.docker.com/build/ci/github-actions/cache/

Note: We should not use Github Action Cache because of its [total image size limit](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy)

Reply:
Done